ext/RMagick/rmagick.h

```diff
@@ -738,6 +765,10 @@ ATTR_ACCESSOR(Image, units)
 ATTR_ACCESSOR(Image, virtual_pixel_method)
 ATTR_ACCESSOR(Image, x_resolution)
 ATTR_ACCESSOR(Image, y_resolution)
+#if !defined(IMAGEMAGICK_7)
+ATTR_ACCESSOR(Image, bias)
+ATTR_ACCESSOR(Image, blur)
```

Comment:
Why did these need to be removed?

Reply:
The `image` class in no longer contains there properties in ImageMagick 7. They are passed as arguments to the methods that need this in ImageMagick 7.

Reply:
What if we were to define methods for them in RMagick with deprecation warnings?

Reply:
They will still be use inside IM6 so we cannot deprecate them. This is one of the things where there will be a different in the API between an IM6 and an IM7 build.

Reply:
Can you share more detail on this? It sounds like there is still a concept of `blur` and `bias`, so maybe there's a way to update it to be backwards compatible.

Reply:
In IM6 the resize/resample method would allow you to overside a `blur` option. This is no longer an option in IM7. But I wonder if this is even being used so we could decide to deprecate this in RMagick if you would prefer that. And the `bias` option has been replaced with a `convolve:bias` option that seems to work in the most recent version of IM6 so that can be removed if you really want that to happen.

Reply:
That seems reasonable. I think we can treat IM7 support as "new feature" so we could potentially do something like:

```c
if (blur_arg) {
#if defined(IMAGEMAGICK_7)
  rb_raise(rb_eArgError, "don't pass blur arg");
#else
  rb_warn("blur arg is deprecated")
#endif
```

Reply:
`blur` has been deprecated so that can be removed. For `bias` we could set/get `convolve:bias` in the property and preserve it. Will modify the PR for this.